app/components/primer/alpha/action_list/item.rb

```diff
@@ -0,0 +1,246 @@
+          @active = active
+          @expanded = expanded
+          @trailing_action_on_hover = trailing_action_on_hover
+          @id = id
+          @system_arguments = system_arguments
+
+          @size = fetch_or_fallback(SIZE_OPTIONS, size, DEFAULT_SIZE)
+          @scheme = fetch_or_fallback(SCHEME_OPTIONS, scheme, DEFAULT_SCHEME)
+          @select_mode = fetch_or_fallback(SELECT_MODE_OPTIONS, select_mode, DEFAULT_SELECT_MODE)
+          @description_scheme = fetch_or_fallback(
+            DESCRIPTION_SCHEME_OPTIONS, description_scheme, DEFAULT_DESCRIPTION_SCHEME
+          )
+
+          @system_arguments[:classes] = class_names(
+            @system_arguments[:classes],
+            SCHEME_MAPPINGS[@scheme],
+            "ActionListItem",
+            # "ActionListItem--withActions",
+            # "ActionListItem--trailingActionHover",
+            # "ActionListItem--withActions" if trailing_action.present? this isn't working,
+            # "ActionListItem--trailingActionHover" => @trailing_action_on_hover, this isn't working
+            "ActionListItem--navActive" => @active,
+            "ActionListItem--subItem" => sub_item?
+          )
+
+          @system_arguments[:role] = role
+
+          @system_arguments[:aria] ||= {}
+          @system_arguments[:aria][:disabled] = "true" if @disabled
+
+          case @select_mode
+          when :single
+            # @system_arguments[:aria][:selected] = "true" if @checked
+          when :multiple
+            # @system_arguments[:aria][:checked] = "true" if @checked
+          end
+
+          @label_arguments = {
+            classes: class_names(
+              "ActionListItem-label",
+              "ActionListItem-label--truncate" => @truncate_label
+            )
+          }
+
+          @content_arguments = {
+            id: @id,
+            classes: class_names(
+              "ActionListContent",
+              SIZE_MAPPINGS[@size]
+            )
+          }
+
+          if @href && !@disabled
+            @content_arguments[:tag] = :a
+            @content_arguments[:href] = @href
+          else
+            @content_arguments[:tag] = :span
+            @content_arguments[:onclick] = on_click if on_click
+          end
+
+          @description_wrapper_arguments = {
+            classes: class_names(
+              "ActionListItem-descriptionWrap",
+              DESCRIPTION_SCHEME_MAPPINGS[@description_scheme]
+            )
+          }
+
+          @sub_group_arguments = {
+            classes: class_names(
+              "ActionList",
+              "ActionList--subGroup"
+            )
+          }
+        end
+
+        def sub_item?
+          @root.present?
+        end
+
+        def expand!
+          # @expanded = true
+        end
+
+        private
+
+        def before_render
+          raise "Cannot render a trailing visual for an item with subitems" if items.present? && trailing_visual.present?
+
+          if items.present?
```

Comment:
@joelhawksley this bit thats commented out is what's missing from the "sub group" logic.